services/convert/pull.go

```diff
@@ -259,3 +261,261 @@ func ToAPIPullRequest(ctx context.Context, pr *issues_model.PullRequest, doer *u
 
 	return apiPullRequest
 }
+
+func ToAPIPullRequests(ctx context.Context, baseRepo *repo_model.Repository, prs issues_model.PullRequestList, doer *user_model.User) ([]*api.PullRequest, error) {
```

Comment:
Is there a test to cover this function?

Reply:
This has been covered by `TestAPIViewPulls` in `tests/integration/api_pull_test.go`.

Reply:
There are so many fields, `api_pull_test.go` seems quite small?

Reply:
I will add more tests.

Reply:
Tests added.